supplemental_content/views.py

```diff
@@ -42,3 +45,38 @@ def get_queryset(self):
                 )
             ).distinct()
         return query
+
+
+class SupplementalContentSectionsView(generics.CreateAPIView):
+    def create(self, request, *args, **kwargs):
+        for section in request.data["sections"]:
+            try:
+                new_section, created = Section.objects.get_or_create(
```

Comment:
We don't need to capture the return variables if we aren't using them.  also, can/should we validate the values in any way.  Is a part of "KNOLL" acceptable?  It not, we should figure out what is acceptable and check it somehow.